<ui version="4.0" >
 <class>MotorTune</class>
 <widget class="QDialog" name="MotorTune" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Motor Tune</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_38" >
       <property name="minimumSize" >
        <size>
         <width>33</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Pwr</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="ui_MotorPower" >
       <property name="minimumSize" >
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_43" >
       <property name="minimumSize" >
        <size>
         <width>33</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Pos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="ui_MotorPosition" >
       <property name="minimumSize" >
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_44" >
       <property name="minimumSize" >
        <size>
         <width>33</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>TPS</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_45" >
       <property name="minimumSize" >
        <size>
         <width>33</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>TTPS</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_46" >
       <property name="minimumSize" >
        <size>
         <width>33</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>P</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="label_47" >
       <property name="minimumSize" >
        <size>
         <width>33</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>I</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="label_48" >
       <property name="minimumSize" >
        <size>
         <width>33</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>33</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="ui_MotorTPS" >
       <property name="minimumSize" >
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLabel" name="ui_MotorTargetTPS" >
       <property name="minimumSize" >
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLabel" name="ui_P" >
       <property name="minimumSize" >
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QLabel" name="ui_I" >
       <property name="minimumSize" >
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="QLabel" name="ui_D" >
       <property name="minimumSize" >
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_33" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Motor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="ui_MotorSelect" >
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum" >
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>PM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ui_PM" >
           <property name="minimumSize" >
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor" >
            <cursorShape>SizeVerCursor</cursorShape>
           </property>
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="value" >
            <number>0</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedControls" >
            <bool>false</bool>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" >
         <item>
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>PD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ui_PD" >
           <property name="minimumSize" >
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedControls" >
            <bool>false</bool>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" >
         <item>
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>IM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ui_IM" >
           <property name="minimumSize" >
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="value" >
            <number>0</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedControls" >
            <bool>false</bool>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" >
         <item>
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ui_ID" >
           <property name="minimumSize" >
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedControls" >
            <bool>false</bool>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" >
         <item>
          <widget class="QLabel" name="label_10" >
           <property name="text" >
            <string>DM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ui_DM" >
           <property name="minimumSize" >
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum" >
            <number>0</number>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="value" >
            <number>0</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedControls" >
            <bool>false</bool>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6" >
         <item>
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>DD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ui_DD" >
           <property name="minimumSize" >
            <size>
             <width>15</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedControls" >
            <bool>false</bool>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="ui_Velocity" >
       <property name="minimum" >
        <number>-1000</number>
       </property>
       <property name="maximum" >
        <number>1000</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
